src/eval-frame/actions/language-actions.js

```diff
@@ -136,7 +152,14 @@ export function runCodeWithLanguage(language, code, messageCallback) {
   if (asyncEvaluator !== undefined) {
     const messageCb =
       messageCallback === undefined ? () => {} : messageCallback;
-    return window[module][asyncEvaluator](code, messageCb);
+    try {
+      return window[module][asyncEvaluator](code, messageCb);
+    } catch (e) {
+      if (e.message === "window[module] is undefined") {
```

Comment:
I think this is a more helpful message than `window[module] is undefined`. If some other error is present, then return that instead.